app/src/main/java/org/simple/clinic/login/applock/AppLockScreen.kt

```diff
@@ -103,7 +121,7 @@ class AppLockScreen(context: Context, attrs: AttributeSet) : RelativeLayout(cont
   fun setIncorrectPinErrorVisible(show: Boolean) {
     when (show) {
       true -> errorTextView.visibility = View.VISIBLE
-      else -> errorTextView.visibility = View.GONE
+      else -> errorTextView.visibility = View.INVISIBLE
```

Comment:
The large gap between the pin text field and forgot-pin button doesn't look very good now that I'm running this new design on my device.